requirements/tests.txt

```diff
@@ -1,7 +1,8 @@
 coveralls==1.5.1
 mock==2.0.0
-mozfile >= 1.1
-mozprocess >= 0.22
+mozfile==1.1
```

Comment:
When you use strict versions, please try to make use of the latest release which currently is 2.0.0 and is Python3 compatible. Please also check the others.